src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketDelete.java

```diff
@@ -103,26 +103,18 @@ public void run(ActionContext context) throws Exception {
           String.format("Unable to access or create bucket %s. ", gcsPath.getBucket())
             + "Ensure you entered the correct bucket path and have permissions for it.", e);
       }
-      gcsPaths.add(new Path(gcsPath.getUri()));
-    }
-
-    FileSystem fs;
-    context.getMetrics().gauge("gc.file.delete.count", gcsPaths.size());
-    for (Path gcsPath : gcsPaths) {
-      try {
-        fs = gcsPaths.get(0).getFileSystem(configuration);
-      } catch (IOException e) {
-        LOG.info("Failed deleting file " + gcsPath.toUri().getPath() + ", " + e.getMessage());
-        // no-op.
-        continue;
-      }
-      if (fs.exists(gcsPath)) {
-        try {
-          fs.delete(gcsPath, true);
-        } catch (IOException e) {
-          LOG.warn(
-            String.format("Failed to delete path '%s'", gcsPath)
-          );
+      String regx = (gcsPath.getName()).replace("*", "[^/]*");
```

Comment:
So we use * to represent any characters right? Can you update the description of this config and doc to explicitly reflect this? There should be a md file related to this plugin

Reply:
Yeah

Reply:
is this regex correct ? "." has special meaning in regex, if user specify `gs://MY_BUCKET/*.csv` , I guess you will match `gs://MY_BUCKET/kklkcvs`